docs/_articles/en/v3.x/salesforce-graph-engine/try-it-yourself.md

```diff
@@ -18,11 +18,9 @@ Next, run a command to view Graph Engine rules, then install our sample project
 To see Graph Engine’s rules, run:
 
 ```bash
-sfdx scanner:rule:list --engine sfge
+sf scanner rule list --engine sfge
 ```
 
-Currently, just one rule is returned. [ApexFlsViolationRule](./en/v3.x/salesforce-graph-engine/rules/#apexflsviolationrule) identifies Create Read Update Delete/Field-Level Security (CRUD/FLS) vulnerabilities in your Apex code.
```

Comment:
@jfeingold35 Really good catch that this sentence is outdated. So I think it's best to remove, because we've said enough: run X to see Graph Engine rules